feat(curriculum): add Artifact Provenance Auditor workshop (#68860)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/blocks/workshop-artifact-provenance-auditor/6a57c331ae94fa4631b60867.md

---
id: 6a57c331ae94fa4631b60867
title: Step 8
challengeType: 1
dashedName: step-8
---

# --description--

> Try your function. Call `getArtifactTitle` with an `id` of `102` and log the result.

Reviewer: ```suggestion
Try your function. Call `getArtifactTitle`, passing `102` as the `id`, and log the result.
```

# --before-each--

```js
const spy = __helpers.spyOn(console, "log");
const getLogs = () => spy.calls.map((call) => call?.[0]);
```

# --hints--

You should call `getArtifactTitle` with `102`.

```js
assert.match(__helpers.removeJSComments(code), /getArtifactTitle\s*\(\s*102\s*\)/);
```

You should log the result of `getArtifactTitle(102)`.

```js

> assert.include(getLogs(), getArtifactTitle(102));

Reviewer: I recommend using a regex here. Otherwise, there is no way to differentiate between the user logging `Bronze Tablet` directly and calling `console.log(getArtifactTitle(102))`.
